### PR TITLE
[#121961157] Re-add format_links filter now that it has been fixed

### PR DESCRIPTION
**Pull request by TheDoubleK**
Work done with @carolinegreen 

Fixes this bug: https://www.pivotaltracker.com/story/show/121961157

Pulls in latest utils with fixed filter, and re-adds the filter to templates where it should be added.

### CODE REVIEW

**Comment by TheDoubleK** [comment]
:+1:

**Comment by carolinegreen** [comment]
👍

File: app/assets/scss/application.scss [hunk 1]
```diff
@@ -158,6 +158,10 @@ $path: "/static/images/";
   }
 }
 
+.break-link {
+  word-break: break-all;
```
Comment (pcraig3) [inline app/assets/scss/application.scss]: Too late now, but I have a suspicion that `break-word` with a fallback to `break-all` might be more elegant.

Comment (carolinegreen) [inline app/assets/scss/application.scss]: I went for break all as some of the links were absolutely horrible and didn't really have words in them, so breaking everything worked better here.

**Pull request merged**